src/util/initial-config.yaml

```diff
@@ -133,7 +133,7 @@ full_node:
   # How often to connect to introducer if we need to learn more peers
   introducer_connect_interval: 500
   # Continue trying to connect to more peers until this number of connections
-  target_peer_count: 10
+  target_peer_count: 15
```

Comment:
Just curious about your thinking here?

Reply:
There are more and more people joining testnet, and our nodes don't have enough space. I just want to ensure that there are enough upnp nodes out there for anyone to join the network. Also more connections =more security, and more load testing.